backends/tfhe-cuda-backend/cuda/src/integer/integer.cu

```diff
@@ -51,17 +51,17 @@ void cuda_full_propagation_64_inplace(
         bsk, lwe_dimension, glwe_dimension, polynomial_size, ks_base_log,
         ks_level, pbs_base_log, pbs_level, grouping_factor, num_blocks);
     break;
-  case 16384:
-    host_full_propagate_inplace<uint64_t, int64_t, AmortizedDegree<16384>>(
-        stream, static_cast<uint64_t *>(input_blocks),
-        (int_fullprop_buffer<uint64_t> *)mem_ptr, static_cast<uint64_t *>(ksk),
-        bsk, lwe_dimension, glwe_dimension, polynomial_size, ks_base_log,
-        ks_level, pbs_base_log, pbs_level, grouping_factor, num_blocks);
-    break;
+//  case 16384:
+//    host_full_propagate_inplace<uint64_t, int64_t, AmortizedDegree<16384>>(
+//        stream, static_cast<uint64_t *>(input_blocks),
+//        (int_fullprop_buffer<uint64_t> *)mem_ptr, static_cast<uint64_t *>(ksk),
+//        bsk, lwe_dimension, glwe_dimension, polynomial_size, ks_base_log,
+//        ks_level, pbs_base_log, pbs_level, grouping_factor, num_blocks);
+//    break;
```

Comment:
I think we should delete those lines instead of commenting them out actually :slightly_smiling_face: Code will be cleaner and it's easy to add them back one day.